src/core/border_router/routing_manager.hpp

```diff
@@ -1489,14 +1493,17 @@ class RoutingManager : public InstanceLocator
         void EvaluateStateChange(State aOldState);
         void WithdrawPrefix(void);
         void StartStop(bool aStart);
+        void PauseResume(bool aPause);
 
         static const char *StateToString(State aState);
 
         using PrefixTimer   = TimerMilliIn<RoutingManager, &RoutingManager::HandlePdPrefixManagerTimer>;
         using StateCallback = Callback<PdCallback>;
 
-        bool          mEnabled;
-        bool          mIsRunning;
+        bool mEnabled;   ///< Whether PdPrefixManager is enabled. (guards the overall PdPrefixManager functions)
+        bool mIsStarted; ///< Whether PdPrefixManager is started. (monitoring prefixes)
+        bool mIsPaused;  ///< Whether PdPrefixManager is paused. (when there is another BR advertising another prefix)
```

Comment:
Remove `///<` which is used for doxygen documentation. We don't want/need to document these in doxysgen docs since they are all private member variables.
```suggestion
        bool mEnabled;   // Whether PdPrefixManager is enabled. (guards the overall PdPrefixManager functions)
        bool mIsStarted; // Whether PdPrefixManager is started. (monitoring prefixes)
        bool mIsPaused;  // Whether PdPrefixManager is paused. (when there is another BR advertising another prefix)
```

Reply:
Done, thanks.